stripe/src/main/java/com/stripe/android/paymentsheet/PaymentResult.kt

```diff
@@ -5,7 +5,10 @@ import android.os.Parcelable
 import com.stripe.android.model.PaymentIntent
 import kotlinx.parcelize.Parcelize
 
-internal sealed class PaymentResult(
+/**
+ * The result a payment sheet operation.
```

Comment:
Are there any special `resultCode` values we can document?

Reply:
This value is actually internal only.